docs/concepts/node-agent.md

```diff
@@ -0,0 +1,59 @@
+# Gardener Node Agent
+
+The goal of the `gardener-node-agent` is to bootstrap a machine into a worker node and maintain node-specific components, which run on the node and are unmanaged by Kubernetes (e.g. the kubelet service, systemd units, ...).
+
+It effectively is a Kubernetes controller deployed onto the worker node.
+
+## Basic Design
+
+This section describes how the `gardener-node-agent` works, what its responsibilities are and how it is installed onto the worker node.
```

Comment:
Maybe this sentence should be ordered in the same way as the whole section. In the sentence installation is the last topic, but in the section it is the first one.
It is somewhat confusing if I start reading and want to know how it works, but get information about how it is installed instead.

Reply:
Great suggestion! I changed the sections and merged Architecture with Basic Design. Now the installation phase is a separate section.